core/controllers/ingest.py

```diff
@@ -80,7 +80,13 @@ def ingest(body, excel_file):
 
     save_submission_file(excel_file, submission_id)
 
-    return "Success: Spreadsheet data ingested", 200
+    return jsonify(
+        {
+            "detail": "Spreadsheet successfully uploaded",
+            "status": 200,
+            "title": "success",
+        }
+    )
```

Comment:
Could you modify the test at `test_ingest.test_ingest_endpoint` to assert against this